### PR TITLE
SNTP client cycle

### PR DESCRIPTION
**Pull request by michalrudolf**
make sntp client cycle

### CODE REVIEW

File: lib/WUI/sntp/sntp_client.c [hunk 1]
```diff
@@ -2,8 +2,8 @@
 #include "sntp_client.h"
 #include "wui_api.h"
 
-static ip_addr_t ntp_server; // testing ntp server located in Prague
-
+static ip_addr_t ntp_server;         // testing ntp server located in Prague
+static bool sntp_processing = false; // describes if sntp is currently running or not
```
Comment (DRracer) [inline lib/WUI/sntp/sntp_client.c:6]: probably `sntp_running` will be a better name

File: lib/WUI/wui_api.h [hunk 2]
```diff
@@ -55,6 +56,9 @@ typedef struct {
     char printer_state[PRI_STATE_STR_LEN]; // state of the printer, have to be set in wui
 } printer_info_t;
 
+extern ETH_STATUS_t eth_status; // discribes state of internet connection
```
Comment (DRracer) [inline lib/WUI/wui_api.h:59]: avoid exposing variables directly - use encapsulating functions instead (like c++ methods)

File: lib/WUI/sntp/sntp_client.c (hunk 1 above)
Comment (michalrudolf) [reply lib/WUI/sntp/sntp_client.c]: commit `e278f27`

File: lib/WUI/wui_api.h (hunk 2 above)
Comment (michalrudolf) [reply lib/WUI/wui_api.h]: commit `e278f27`

**Pull request merged**